.github/workflows/lint_and_test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Lint package
-        uses: golangci/golangci-lint-action@v5
```

Comment:
@alexandear  - Looks like v6.1.1 might be a better version to go with here; is there a reason to go with v6 instead?

Reply:
Ah, gotcha; makes sense.

I'll get this merged then :)